test/common/filter/http/filter_config_discovery_impl_test.cc

```diff
@@ -51,6 +52,7 @@ class FilterConfigDiscoveryTestBase : public testing::Test {
     ON_CALL(init_manager_, initialize(_))
         .WillByDefault(Invoke(
             [this](const Init::Watcher& watcher) { init_target_handle_->initialize(watcher); }));
+    EXPECT_CALL(factory_context_.admin_, concurrency()).WillRepeatedly(Return(0));
```

Comment:
nit: can you just make this a nice mock?

Reply:
It is a nice mock but it defaults to 1 for integration test purposes. Having value 0 is weird but it is what it is in the unit tests.

Reply:
OK can you add a comment and switch this to ON_CALL/WillByDefault?